crates/provider/Cargo.toml

```diff
@@ -67,6 +67,7 @@ alloy-transport-http = { workspace = true, features = ["reqwest"] }
 
 itertools.workspace = true
 reqwest.workspace = true
+serde_json = { workspace = true, features = ["preserve_order"] }
```

Comment:
please undo, we dont want to enable this because this can cause a ton of side-effects